artiq/test/test_rpctool.py

```diff
@@ -0,0 +1,38 @@
+import os, sys
+import asyncio
+import unittest
+from artiq.protocols.pc_rpc import Server
+
+class Target:
+    def output_value(self):
+        return 4125380
+
+class rpctool_test(unittest.TestCase):
+    async def check_value(self):
+        proc = asyncio.create_subprocess_shell(
+                            sys.executable + " -m artiq.frontend.artiq_rpctool ::1 3249 call output_value", 
```

Comment:
This will fail if ``sys.executable`` contains a space character. Better pass the arguments with a list and use ``create_subprocess_exec`` as done elsewhere.

Reply:
Also, 3249 is already used by another ARTIQ service. Better pick another port, e.g. 7777 as used in test_pc_rpc.py.

Reply:
I get this when using` subprocess_exec`: `[WinError 2] The system cannot find the file specified`, which happened the same previously when I use `Popen` without adding `shell=True`.
Also, only string arguments are accepted. Can I simply replace `sys.executable` by `python`?

Reply:
What is the complete code you were using? It should be with multiple arguments, something like ``create_subprocess_exec(sys.executable, "-m", "artiq.frontend.artiq_rpctool", ...)`` and not a single string.